jetty-ee10/jetty-ee10-osgi/test-jetty-ee10-osgi/src/test/java/org/eclipse/jetty/ee10/osgi/test/TestOSGiUtil.java

```diff
@@ -132,39 +132,17 @@ public static void coreJettyDependencies(List<Option> res)
             res.add(systemProperty("org.ops4j.pax.url.mvn.settings").value(System.getProperty("settingsFilePath")));
         }
 
-        res.add(mavenBundle().groupId("org.slf4j").artifactId("slf4j-api").versionAsInProject().noStart());
-
-        /*
-         * Jetty 11 uses slf4j 2.0.0 by default, however we want to test with slf4j 1.7.30 for backwards compatibility.
-         * To do that, we need to use slf4j-simple as the logging implementation. We make a simplelogger.properties
-         * file available so that jetty logging can be configured
-         */
-        // BEGIN - slf4j 1.7.x
-        /* slf4j-simple conflicts with both slf4j 1.7.x, and jetty-slf4j-impl. (but in different ways)
-
-        TinyBundle simpleLoggingPropertiesBundle = TinyBundles.bundle();
-        simpleLoggingPropertiesBundle.add("simplelogger.properties", ClassLoader.getSystemResource("simplelogger.properties"));
-        simpleLoggingPropertiesBundle.set(Constants.BUNDLE_SYMBOLICNAME, "simple-logger-properties");
-        simpleLoggingPropertiesBundle.set(Constants.FRAGMENT_HOST, "slf4j-simple");
-        simpleLoggingPropertiesBundle.add(FragmentActivator.class);
-        res.add(CoreOptions.streamBundle(simpleLoggingPropertiesBundle.build()).noStart());
-        res.add(mavenBundle().groupId("org.slf4j").artifactId("slf4j-simple").versionAsInProject().noStart());
-         */
-        // END - slf4j 1.7.x
-
-        /*
-         * When running with slf4j >= 2.0.0, remove the slf4j simple logger above and uncomment the following lines
-         */
-        // BEGIN - slf4j 2.x
+        /* SLF4J 2.0 */
+        // slf4j-api comes from paxexam now.
+        // res.add(mavenBundle().groupId("org.slf4j").artifactId("slf4j-api").versionAsInProject().noStart());
```

Comment:
it's probably not sync with ee9 :( 
btw I noticed some issues recently and logging was a pain...
I'd like to extract some parts here which can be common and if possible really fix the logging

Reply:
@janbartel sure but my point is either we don't change anything in this PR for this or we keep both files in sync (because something tell me we will forget :) ) BTW they are probably already not in sync anymore.